static/app/views/replays/list/replays.tsx

```diff
@@ -50,13 +50,13 @@ function ReplaysList() {
     organization,
   });
 
-  const {hasSentOneReplay} = useReplayOnboardingSidebarPanel();
+  const {hasSentOneReplay, fetching} = useHaveSelectedProjectsSentAnyReplayEvents();
 
   return (
     <Layout.Body>
       <Layout.Main fullWidth>
         <ReplaysFilters />
-        {hasSentOneReplay ? (
+        {fetching ? null : hasSentOneReplay ? (
```

Comment:
No loader?

Reply:
I looked at a `<Placeholder>` but the table is variable height, it felt like more flashing in and out to my eyes